app/views/comments/_answercomments.html.erb

```diff
@@ -1,16 +1,18 @@
 <div id="answer-comments" class="blue-text">
   <% answer.comments.each do |comment| %>
+  <div class="card-panel">
     <div id="answer-comment-body" style= padding-left:50px;>
       <p><%= comment.body %></p>
       <div id='delete-comment-button' class='left card-action'>
-        <% if current_admin? %>
-          <%= button_to "Delete Comment", comment_path(comment), method: :delete, class: "waves-effect red darken-4 waves-light btn-small" %>
+        <% if current_admin? || current_users_comment?(comment)%>
```

Comment:
Given that this is repeated, should we consider extracting it to it's own method somewhere?